libdirections-session/src/main/java/com/mapbox/navigation/directions/session/MapboxDirectionsSession.kt

```diff
@@ -4,12 +4,21 @@ import com.mapbox.annotation.navigation.module.MapboxNavigationModule
 import com.mapbox.annotation.navigation.module.MapboxNavigationModuleType
 import com.mapbox.geojson.Point
 import com.mapbox.navigation.base.route.DirectionsSession
-import com.mapbox.navigation.base.route.Route
 import com.mapbox.navigation.base.route.Router
+import com.mapbox.navigation.base.route.model.Route
 import java.util.concurrent.CopyOnWriteArrayList
 
 @MapboxNavigationModule(MapboxNavigationModuleType.DirectionsSession, skipConfiguration = true)
-class MapboxDirectionsSession(private val router: Router) : DirectionsSession {
+class MapboxDirectionsSession(
+    private val router: Router,
+    origin: Point,
+    waypoints: List<Point>,
+    private val destination: Point
```

Comment:
This should be a `var` and have a setter method (that makes a request) exposed like other variables.

Reply:
done